pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/immutable/ImmutableSegmentImpl.java

```diff
@@ -213,6 +215,16 @@ public List<StarTreeV2> getStarTrees() {
     return _starTreeIndexContainer != null ? _starTreeIndexContainer.getStarTrees() : null;
   }
 
+  @Nullable
+  public void setValidDocSnapshots(ImmutableRoaringBitmap validDocSnapshots) {
+    _validDocSnapshots = validDocSnapshots;
+  }
+
+  @Nullable
+  public ImmutableRoaringBitmap getValidDocSnapshots() {
```

Comment:
(minor) Suggest renaming it to `getValidDocIdsSnapshot()`

Reply:
gotcha, thanks